workers/unity/Packages/com.improbable.gdk.core/Config/LocatorConfig.cs

```diff
@@ -37,11 +38,29 @@ public LocatorConfig()
         /// </summary>
         public override void Validate()
         {
-            ValidateConfig(LocatorHost, RuntimeConfigNames.LocatorHost);
+            ValidateString(LocatorHost, RuntimeConfigNames.LocatorHost);
+            ValidateString(LocatorParameters.ProjectName, RuntimeConfigNames.ProjectName);
+            ValidateCondition(
+                LocatorParameters.CredentialsType == LocatorCredentialsType.LoginToken ||
+                LocatorParameters.CredentialsType == LocatorCredentialsType.Steam,
+                "LocatorConfig has invalid CredentialsType.");
```

Comment:
What should a valid credentialstype be?

Reply:
i.e. how can the developer resolve this?

Reply:
they need to check their command line arguments and fix them

Reply:
Since we ignore a member within the enum let's mention in this comment it can only be this or that

Reply:
I've refined the error message so that it's easier to tell what needs to be done to fix the problem.